src/config/pci-passthrough.md

```diff
@@ -0,0 +1,405 @@
+        00:04.0 Host bridge [0600]: Advanced Micro Devices, Inc. [AMD] Family 17h (Models 00h-1fh) PCIe Dummy Host Bridge [1022:1452] 
+IOMMU Group 7: 
+        00:07.0 Host bridge [0600]: Advanced Micro Devices, Inc. [AMD] Family 17h (Models 00h-1fh) PCIe Dummy Host Bridge [1022:1452] 
+IOMMU Group 8: 
+        00:07.1 PCI bridge [0604]: Advanced Micro Devices, Inc. [AMD] Family 17h (Models 00h-0fh) Internal PCIe GPP Bridge 0 to Bus B [1022:1454] 
+IOMMU Group 9: 
+        00:08.0 Host bridge [0600]: Advanced Micro Devices, Inc. [AMD] Family 17h (Models 00h-1fh) PCIe Dummy Host Bridge [1022:1452] 
+```
+
+I bought my motherboard specifically for PCI passthrough so my IOMMU group are
+very neat. In fact, I can passthrough some of my USB controllers while keeping
+the rest of the motherboard's peripherals on my host system. If I passthrough
+group 13 which has my host graphics card (r9 270x), I would also have to
+passthrough the other components in that group to my guest. As such, passing
+through my host graphics card would result in me passing through the following
+components:
+
+```
+IOMMU Group 13: 
+        01:00.0 USB controller [0c03]: Advanced Micro Devices, Inc. [AMD] X370 Series Chipset USB 3.1 xHCI Controller [1022:43b9] (rev 02) 
+        01:00.1 SATA controller [0106]: Advanced Micro Devices, Inc. [AMD] X370 Series Chipset SATA Controller [1022:43b5] (rev 02) 
+        01:00.2 PCI bridge [0604]: Advanced Micro Devices, Inc. [AMD] X370 Series Chipset PCIe Upstream Port [1022:43b0] (rev 02) 
+        02:00.0 PCI bridge [0604]: Advanced Micro Devices, Inc. [AMD] 300 Series Chipset PCIe Port [1022:43b4] (rev 02) 
+        02:02.0 PCI bridge [0604]: Advanced Micro Devices, Inc. [AMD] 300 Series Chipset PCIe Port [1022:43b4] (rev 02) 
+        02:03.0 PCI bridge [0604]: Advanced Micro Devices, Inc. [AMD] 300 Series Chipset PCIe Port [1022:43b4] (rev 02) 
+        02:04.0 PCI bridge [0604]: Advanced Micro Devices, Inc. [AMD] 300 Series Chipset PCIe Port [1022:43b4] (rev 02) 
+        03:00.0 USB controller [0c03]: ASMedia Technology Inc. ASM1143 USB 3.1 Host Controller [1b21:1343] 
+        04:00.0 Ethernet controller [0200]: Intel Corporation I211 Gigabit Network Connection [8086:1539] (rev 03) 
+        05:00.0 Ethernet controller [0200]: Qualcomm Atheros Killer E2500 Gigabit Ethernet Controller [1969:e0b1] (rev 10) 
+```
+
+This is why the PCI slot you plug in your graphics card matters. Some PCI slots
+may have other peripherals in the same IOMMU group that makes the slot
+inconvienient to pass through. Once you have tweaked this and are fine with any
+extra peripherals that may be unusable on your host, continue to the next step.
+
+I will be passing through group 14.
+
+### Isolating GPU
+
+Find the GPU *and its audio device* you want to pass through to your guest in
+IOMMU group list and get the content inside the second square brackets.
+
+For example the GPU I want to passthrough and its audio device had the following
```

Comment:
Not sure this example, from this line through to L294, are required. Instead, suggest changing L282 to "the second square brackets, e.g. `1002:67df`.